n2y/converter.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import annotations
```

Comment:
What is this import for?

Reply:
That is vscode trying to be helpful. Have to watch out for these. Please delete @sHermanGriffiths.

In vscode if you try to use a variable without first assigning it, vscode will try to find something that it can import to match that name. This is both helpful and annoying. Vscode is awesome for some things, but not as smart as PyCharm when it comes to python.

Reply:
deleted